packages/db/schema.ts

```diff
@@ -20,6 +20,7 @@ import {
 	pgEnum,
 	primaryKey,
 	pgTable,
+	serial,
```

Comment:
Question: Was serial meant to be kept here? Seems like it was added and then never used.

Reply:
That one slipped through the cracks when I had temporarily added a serial pk to users; this will be removed.